src/components/SocialBar/SocialBarIcon.st.css

```diff
@@ -0,0 +1,18 @@
+
+.root {
+
+}
+
+.iconButton {
+  -st-states: theme(enum(light, dark));
+}
+
+.iconButton:theme(light) path {
+  fill: black !important;
```

Comment:
Is the `!important` necessary here?

Reply:
after changing selector to `.root:theme(light) .iconButton path` not necessary now.